packages/cfd/src/Containers/trade-modal.tsx

```diff
@@ -0,0 +1,409 @@
+        case 'ctrader':
+            return getPlatformCTraderDownloadLink(type);
+        case 'derivez':
+            return getPlatformDerivEZDownloadLink(type);
+        default:
+            return getPlatformDXTradeDownloadLink(type);
+    }
+};
+
+const getTitle = (market_type: string, is_eu_user: boolean) => {
+    if (is_eu_user) localize('MT5 CFDs');
+    return market_type;
+};
+
+const DxtradeDesktopDownload = ({ dxtrade_tokens, is_demo }: TDxtradeDesktopDownloadProps) => {
+    return (
+        <React.Fragment>
+            <a
+                className='cfd-trade-modal__dxtrade-button'
+                href={getDXTradeWebTerminalLink(
+                    is_demo ? 'demo' : 'real',
+                    dxtrade_tokens && dxtrade_tokens[is_demo ? 'demo' : 'real']
+                )}
+                target='_blank'
+                rel='noopener noreferrer'
+            >
+                <Icon className='cfd-trade-modal__dxtrade-button-icon' icon='IcBrandDxtradeWordmark' size={36} />
+                <div className='cfd-trade-modal__dxtrade-button-text'>
+                    <Text color='colored-background' size='xxs' weight='bold'>
+                        <Localize i18n_default_text='Web terminal' />
+                    </Text>
+                </div>
+            </a>
+        </React.Fragment>
+    );
+};
+
+const DerivEZDesktopDownload = () => {
+    return (
+        <React.Fragment>
+            <a className='cfd-trade-modal__dxtrade-button' href={''} target='_blank' rel='noopener noreferrer'>
+                <Icon className='cfd-trade-modal__dxtrade-button-icon' icon='IcBrandDerivEzWordmark' size={36} />
+                <div className='cfd-trade-modal__dxtrade-button-text'>
+                    <Text color='colored-background' size='xxs' weight='bold'>
+                        <Localize i18n_default_text='Web terminal' />
+                    </Text>
+                </div>
+            </a>
+        </React.Fragment>
+    );
+};
+
+const CTraderDesktopDownload = () => {
+    return (
+        <React.Fragment>
+            <a
+                className='cfd-trade-modal__dxtrade-button'
+                href={'https://ct.deriv.com/'}
+                target='_blank'
+                rel='noopener noreferrer'
+            >
+                <Icon className='cfd-trade-modal__dxtrade-button-icon' icon='IcBrandCtraderWordmark' size={36} />
+                <div className='cfd-trade-modal__dxtrade-button-text'>
+                    <Text color='colored-background' size='xxs' weight='bold'>
+                        <Localize i18n_default_text='Web terminal' />
+                    </Text>
+                </div>
+            </a>
+        </React.Fragment>
+    );
+};
+
+const TradeModal = ({
+    mt5_trade_account,
+    is_eu_user,
+    onPasswordManager,
+    toggleModal,
+    dxtrade_tokens,
+    is_demo,
+    platform,
+}: TradeModalProps) => {
+    const differentQRCodes = (acc_type: TPlatformType) => {
```

Comment:
Let's move this to a config file as a helper function and pls rename to something more descriptive e.g. `getPlatformQRCode`